### ISSUE TITLE
Parallel DN can no longer auto-populate nodefiles using --out with --in_file_list

### ISSUE THREAD

**Issue by zarzycki** [ClimateGlobalChange/tempestextremes#25]
Previous workflow takes --in_file_list of N files and then uses --out to specify an out nodefile prefix that is appended with the line number of the process (e.g., cyclones.OUT.001, cyclones.OUT.002, etc.)

These are then catted (e.g., cat cyclones.OUT* > fullnodefile) before passing the candidates into SN.

Error thrown is:

..EXCEPTION (/glade/work/zarzycki/tempestextremes/src/nodes/DetectNodes.cpp, Line 1365) Assertion failure

**Comment by zarzycki** [comment]
Commenting out line 1365 in DN (assertion) allows DN to run with MPI and autopopulates N out files as before, but all files are empty.

Each log file shows the following lines, with no other information

```
Generating RLL grid data
..Total calculated grid area: 1.256637061435794e+01
..Done
```

**Comment by paullric** [comment]
Should be fixed in 3c02c05561966c62e19ba4b5d3860b38728dc27b.  --timefilter was triggering when it wasn't specified, causing essentially all times to be filtered out.

**Comment by zarzycki** [comment]
Can confirm 3c02c05 now BFB reproduces CFSR tracks with existing scripts.
